decidim-core/config/routes.rb

```diff
@@ -5,6 +5,9 @@
 
   get "/offline", to: "offline#show"
 
+  post "/subscribe_to_notifications", to: "notifications_subscriptions#create"
```

Comment:
I think we should apply the Rails conventions here normally:
```ruby
resources :notification_subscriptions, only: [:create, :delete]
```

Or is there some particular reason why these are two separate routes?

Reply:
I did that way because I found it more descriptive. But maybe it's better to follow the REST convention. I change it

Reply:
I saw that we have this unresolved conversation @ferblape, can you apply the changes mentioned by @ahukkanen? (Sorry that I didn't see it before!)